build-plugin/action.yml

```diff
@@ -0,0 +1,148 @@
+name: "Grafana Build Plugin"
+description: "Builds a Grafana plugin"
+
+inputs:
+  token:
+    description: "Token for the repository. Can be passed in using `{{ secrets.GITHUB_TOKEN }}`."
+    required: false
+    default: "${{ github.token }}"
+  policy_token:
+    description: "Grafana access policy token. https://grafana.com/developers/plugin-tools/publish-a-plugin/sign-a-plugin#generate-an-access-policy-token"
+    required: false
+    default: ""
+  grafana_token:
+    description: "[deprecated] Grafana API Key to sign a plugin. Prefer `policy_token`."
+    required: false
+    default: ""
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 20
+
+    - name: Setup Go environment
+      uses: actions/setup-go@v3
+      with:
+        go-version: "1.21"
+
+    - name: Install dependencies
+      run: ${{ github.action_path }}/pm.sh install
+      shell: bash
+
+    - name: Build and test frontend
+      run: ${{ github.action_path }}/pm.sh build
+      shell: bash
+
+    - name: Check for backend
+      id: check-for-backend
+      run: |
+        if [ -f "Magefile.go" ]
+        then
+          echo "has-backend=true" >> $GITHUB_OUTPUT
+        fi
+      shell: bash
+
+    - name: Test backend
+      if: steps.check-for-backend.outputs.has-backend == 'true'
+      uses: magefile/mage-action@v2
+      with:
+        version: latest
+        args: coverage
+
+    - name: Build backend
+      if: steps.check-for-backend.outputs.has-backend == 'true'
+      uses: magefile/mage-action@v2
+      with:
+        version: latest
+        args: buildAll
+
+    - name: Warn missing Grafana access policy token
+      run: |
+        echo Please generate a Grafana access policy token: https://grafana.com/developers/plugin-tools/publish-a-plugin/sign-a-plugin#generate-an-access-policy-token
+        echo Once done please follow the instructions found here: https://github.com/${{github.repository}}/blob/main/README.md#using-github-actions-release-workflow
```

Comment:
```suggestion
        echo Once done, follow the instructions found here: https://github.com/${{github.repository}}/blob/main/README.md#using-github-actions-release-workflow
```

Reply:
Prefer to avoid "please" except when the user has been inconvenienced. If used once, prefer to limit to one "please" per page.